RadarSDK/RadarLocationManager.m

```diff
@@ -192,8 +195,6 @@ - (void)startUpdates:(int)interval {
                                                        block:^(NSTimer *_Nonnull timer) {
                                                            [[RadarLogger sharedInstance] logWithLevel:RadarLogLevelDebug message:@"Timer fired"];
 
-                                                           [[RadarBackgroundTaskManager sharedInstance] startBackgroundTask];
```

Comment:
I know we're removing this to cleanup log messages that are coming out, how confident are we that this is no longer needed for the purpose of get location updates?

Reply:
You used to need to start background tasks to get (and keep extending) 180 seconds of background runtime, but as long as you're also constantly running a location manager, this seemingly stopped being necessary somewhere around iOS 10 (not documented by Apple, just something I observed). What's worse, there's now a scary warning message if you run a background task longer than 30 seconds (which @jsani-radar noted when dogfooding). We left it in to be on the safe side, but I've been testing for a few days without it and I've never seen background time run out, so I think we're safe to remove